mocks/helpers.go

```diff
@@ -104,3 +83,83 @@ func NewMockStub(t *testing.T) *ChaincodeStub {
 
 	return mockStub
 }
+
+// GetNewStringNonce returns string value of nonce based on current time
+func GetNewStringNonce() string {
+	return strconv.FormatInt(time.Now().UnixNano()/1000000, 10)
+}
+
+// SetFunctionAndParameters sets function name and its parameters to ChaincodeStub
+func SetFunctionAndParameters(mockStub *ChaincodeStub, functionName, requestID, channelName, chaincodeName string, args ...string) {
+	ctorArgs := append(append([]string{requestID, channelName, chaincodeName}, args...), GetNewStringNonce())
+	mockStub.GetFunctionAndParametersReturns(functionName, ctorArgs)
+}
+
+// SetFunctionAndParametersWithSign sets function name and parameters with sender sign to ChaincodeStub
+func SetFunctionAndParametersWithSign(
+	mockStub *ChaincodeStub,
+	user *UserFoundation,
+	functionName,
+	requestID,
+	channelName,
+	chaincodeName string,
+	args ...string,
+) error {
+	ctorArgs := append(append([]string{functionName, requestID, channelName, chaincodeName}, args...), GetNewStringNonce())
+
+	pubKey, sMsg, err := user.Sign(ctorArgs...)
+	if err != nil {
+		return err
+	}
+
+	mockStub.GetFunctionAndParametersReturns(functionName, append(ctorArgs[1:], pubKey, base58.Encode(sMsg)))
+	return nil
+}
+
+// ACLGetAccountInfo mocks positive response from ACL
+func ACLGetAccountInfo(t *testing.T, mockStub *ChaincodeStub, invokeCallCount int) {
+	accInfo := &pbfound.AccountInfo{
+		GrayListed:  false,
+		BlackListed: false,
+	}
```

Comment:
The GrayListed and BlackListed parameters can be skipped, they will still be false